study-group/Toast-Implementation-Studies.md

```diff
@@ -79,8 +79,121 @@ The `restoreFocusOnClear` attribute restores page's focus once the toast is clea
     - How to properly handle top layer is an ongoing debate (see [here](https://github.com/whatwg/html/issues/897#issuecomment-198512778))
 
 ## [Material-UI Snackbar](https://material-ui.com/api/snackbar/)
+Material-UI Snackbar is a component from Google's Material-UI React library that provides an interactive toast notification.
+
+### Design & Sample Code
+"Snackbar" is the name used in many Google component libraries for an extensible, interactable toast component.
+Unlike Google *Toasts*, the Snackbar can have a call to action and a close button, alongside other differentiating properties.
```

Comment:
semantic breaks for this file too. If you're wondering how aggressively to break things, take a look at

https://github.com/w3c/csswg-drafts/blob/master/css-shadow-parts-1/Overview.bs

Reply:
On it